src/components/DevfileEditor/index.tsx

```diff
@@ -243,8 +244,8 @@ export class DevfileEditor extends React.PureComponent<Props, State> {
     );
   }
 
-  private getDecorations(): monaco.editor.IModelDecoration[] {
-    const decorations: monaco.editor.IModelDecoration[] = [];
+  private getDecorations(): any[] {
+    const decorations: any[] = [];
```

Comment:
Please take this into account: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#any

Reply:
Good catch. I forgot to change back. I have fixed it.